src/lib/Libifl/pbsD_deljob2.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 1994-2020 Altair Engineering, Inc.
+ * For more information, contact Altair at www.altair.com.
+ *
+ * This file is part of both the OpenPBS software ("OpenPBS")
+ * and the PBS Professional ("PBS Pro") software.
+ *
+ * Open Source License Information:
+ *
+ * OpenPBS is free software. You can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * OpenPBS is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Commercial License Information:
+ *
+ * PBS Pro is commercially licensed software that shares a common core with
+ * the OpenPBS software.  For a copy of the commercial license terms and
+ * conditions, go to: (http://www.pbspro.com/agreement.html) or contact the
+ * Altair Legal Department.
+ *
+ * Altair's dual-license business model allows companies, individuals, and
+ * organizations to create proprietary derivative works of OpenPBS and
+ * distribute them - whether embedded or bundled with other software -
+ * under a commercial license agreement.
+ *
+ * Use of Altair's trademarks, including but not limited to "PBS™",
+ * "OpenPBS®", "PBS Professional®", and "PBS Pro™" and Altair's logos is
+ * subject to Altair's trademark licensing policies.
+ */
+
+/**
+ * @file	pbs_deljob.c
+ * @brief
+ * Send the Delete Job request to the server
+ * really just an instance of the manager request
+ */
+
+#include <pbs_config.h>   /* the master config generated by configure */
+
+#include <stdio.h>
+#include "libpbs.h"
+
+
+/**
+ * @brief
+ *	Send the Delete Job request to the server
+ * 	really just an instance of the manager request
+ *
+ * @param[in] c - connection handler
+ * @param[in] jobid - job identifier
+ * @param[in] extend - string to encode req
+ *
+ * @return	struct batch_status *
+ * @retval	0	success
+ * @retval	!0	error
+ *
+ */
+
+struct batch_deljob_status *
+__pbs_deljoblist(int c, char **jobid, char *extend)
+{
+	struct attropl *aoplp = NULL;
+
+	if ((jobid == NULL) || (**jobid == '\0'))
+		return NULL;
+
+	return PBSD_delete(c, PBS_BATCH_DeleteJobList,
```

Comment:
Do you need separate files for int_delete2.c and int_deleteput.c? Can we keep those functions within pbsD_deljob2?